pkg/sendbuffer/sendbuffer.go

```diff
@@ -149,6 +149,11 @@ func (sb *SendBuffer) Run(ctx context.Context) error {
 		case <-sb.sendTicker.C:
 			continue
 		case <-ctx.Done():
+			// Send one final batch, if possible, so that we can get logs related to shutdowns
+			time.Sleep(1 * time.Second)
```

Comment:
why the sleep? Just to give any other logs a chance to catch up? can you put a comment here please?

Reply:
Yup exactly! Added a comment, thank you

Reply:
This is certainly simpler than adding in shutdown ordering to the rungroup. Well, a developer can dream...